css/amp.css

```diff
@@ -219,10 +219,10 @@ i-amphtml-sizer {
  * does not allow activation of buttons.
  */
 .i-amphtml-screen-reader {
-  position: fixed !important;
-  /* keep it on viewport */
-  top: 0px !important;
-  left: 0px !important;
+  position: absolute !important;
+  /* keep it on viewport. TalkBack likes this >=12px */
+  top: 12px !important;
```

Comment:
huh, even if top:0 is still visible and not hidden behind some other random thing? I am worried if it is amp-carousel that has some margin making a top:0 position absolute child hidden somehow. Could you verify?